com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettings.cs

```diff
@@ -15,45 +15,45 @@ public class DebugDisplaySettings : IDebugDisplaySettingsQuery
         /// </summary>
         public static DebugDisplaySettings Instance => s_Instance.Value;
 
-        DebugDisplaySettingsCommon CommonSettings { get; set; }
+        DebugDisplaySettingsCommon commonSettings { get; set; }
 
         /// <summary>
         /// Material-related Rendering Debugger settings.
         /// </summary>
-        internal DebugDisplaySettingsMaterial MaterialSettings { get; private set; }
+        public DebugDisplaySettingsMaterial materialSettings { get; private set; }
 
         /// <summary>
         /// Rendering-related Rendering Debugger settings.
         /// </summary>
-        internal DebugDisplaySettingsRendering RenderingSettings { get; private set; }
+        public DebugDisplaySettingsRendering renderingSettings { get; private set; }
 
         /// <summary>
         /// Lighting-related Rendering Debugger settings.
         /// </summary>
-        internal DebugDisplaySettingsLighting LightingSettings { get; private set; }
+        public DebugDisplaySettingsLighting lightingSettings { get; private set; }
 
         #region IDebugDisplaySettingsQuery
 
         /// <summary>
         /// Returns true if any of the debug settings are currently active.
         /// </summary>
-        public bool AreAnySettingsActive => MaterialSettings.AreAnySettingsActive ||
-        LightingSettings.AreAnySettingsActive ||
-        RenderingSettings.AreAnySettingsActive;
+        public bool AreAnySettingsActive => materialSettings.AreAnySettingsActive ||
+        lightingSettings.AreAnySettingsActive ||
+        renderingSettings.AreAnySettingsActive;
 
         public bool TryGetScreenClearColor(ref Color color)
         {
-            return MaterialSettings.TryGetScreenClearColor(ref color) ||
-                RenderingSettings.TryGetScreenClearColor(ref color) ||
-                LightingSettings.TryGetScreenClearColor(ref color);
+            return materialSettings.TryGetScreenClearColor(ref color) ||
+                renderingSettings.TryGetScreenClearColor(ref color) ||
+                lightingSettings.TryGetScreenClearColor(ref color);
         }
 
         /// <summary>
         /// Returns true if lighting is active for current state of debug settings.
         /// </summary>
-        public bool IsLightingActive => MaterialSettings.IsLightingActive &&
-        RenderingSettings.IsLightingActive &&
-        LightingSettings.IsLightingActive;
+        public bool IsLightingActive => materialSettings.IsLightingActive &&
+        renderingSettings.IsLightingActive &&
+        lightingSettings.IsLightingActive;
```

Comment:
It wont be better to iterate over `m_Settings`?, in this way any time we add another `IDebugDisplaySettingsData`. We do not need to touch this properties, and methods

Reply:
Fair point, I've changed it to do as you suggested.